packages/core/src/predicate.ts

```diff
@@ -43,6 +43,28 @@ const predicates: Record<string, PredicateFactory> = {
       }
       return result
     }
+  },
+
+  $gt: (query: number, propertyKey: string): Predicate => {
+    return (docs: Doc[]): Doc[] => {
+      const result: Doc[] = []
+      for (const doc of docs) {
+        const value = (doc as any)[propertyKey]
+        if (value - query > 0) result.push(doc)
```

Comment:
Should it be just for number formats?

Reply:
Now, yes, I can hardly imagine a situation when you need `gt` not by the number ...